CHANGELOG.md

```diff
@@ -16,6 +16,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.10.0] - 2023-06-17
```

Comment:
There's a section at the end of the file that contains the links that also need to be updated.

Reply:
oh, and it's currently 2024 :smile:

Reply:
Whoops - Thanks! 😁